Modules/_xxtestfuzz/fuzzer.c

```diff
@@ -88,6 +88,14 @@ static int _run_fuzz(const uint8_t *data, size_t size, int(*fuzzer)(const char*
 /* CPython generates a lot of leak warnings for whatever reason. */
 int __lsan_is_turned_off(void) { return 1; }
 
+wchar_t wide_program_name[NAME_MAX];
+
+int LLVMFuzzerInitialize(int *argc, char ***argv) {
```

Comment:
Did you mean `**argv`?

Reply:
LLVM passes both `argc` and `argv` as pointers so the fuzzer can mutate them: https://llvm.org/docs/LibFuzzer.html#startup-initialization

Reply:
Oh, I didn't know that. Even today I've learned something useful. Thank you very much. :)